LICENSE.AGPL

```diff
@@ -0,0 +1,661 @@
+                    GNU AFFERO GENERAL PUBLIC LICENSE
```

Comment:
Sorry if my comment wasn't really clear. One problem I saw is that less people might actually participate and modify the code in the official repo (for instance for adding new protocols) because of the CLA (and people would rather fork and build their own modified cothority instead; not sure if this is good or bad). But, I guess, I was barking up the wrong tree with my concerns (learned a new idiom ;-). 

>Yes, it would be nice if the code can run different protocols without modifying the code, but that's an orthogonal (technical) issue we should discuss separately.

Would be happy to discuss this issue further (maybe offline). I think we all (?) weren't aware that the "extension message/protocol" that uses ASCII as you described it would allow new protocols, ideally, without modifying the code. I'm not sure this is possible with the current architecture.

Thanks again!